### PR TITLE
Fixes #9466 - gutterball.conf missing gutterball.amqp.connect

### PR DESCRIPTION
**Pull request by dustints**

### CODE REVIEW

File: templates/gutterball.conf.erb [hunk 1]
```diff
@@ -10,5 +10,6 @@ jpa.config.hibernate.show_sql=false
 # password comes from /etc/pki/katello/keystore_password-file
 gutterball.amqp.keystore_password=<%= keystore_password %>
 gutterball.amqp.truststore_password=<%= keystore_password %>
+gutterball.amqp.connect=amqp://guest:guest@<%= @fqdn %>/test:5671?brokerlist='tcp://<%= @fqdn %>:5671'?ssl='true'&ssl_cert_alias='gutterball'
```
Comment (thomasmckay) [inline templates/gutterball.conf.erb]: gutterball.amqp.connect=amqp://guest:guest@<%= @fqdn %>/test:5671?brokerlist='tcp://<%= @fqdn %>:5671?ssl='true'&ssl_cert_alias='gutterball''

Check ', I corrected above.

Is port 5671 always that or is it configurable somewhere? (ie. is hardcode alright?)

Comment (dustints) [inline templates/gutterball.conf.erb]: @thomasmckay ur right, broker hostname and port should be made configurable;
it's set in the qpid module.

Comment (dustints) [inline templates/gutterball.conf.erb]: @thomasmckay  updated

**Comment by thomasmckay** [comment]
ack

**Comment by dustints** [comment]
waiting for happy lint

**Pull request merged**